index.html

```diff
@@ -98,10 +98,55 @@ <h2 class="work-title">tonic</h2>
           <li>Javascript</li>
         </ul>
         <div>
-          <button type="button" class="btn">See Project</button>
+          <button data-modal-target="#modal" class="btn">See Project</button>
+
+          <div class="modal" id="modal">
+
+            <div class="modal-description">
+              <h2 class="work-title">tonic</h2>
+              <button data-btn-close class="btn-close">&times;</button>
+            </div>
+
+            <div class="modal-body">
+              <ul class="canopy">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
```

Comment:
- [ ] As suggested by the previous reviewer, the background of the popup should be blurred on both mobile and desktop devices, as depicted in the Figma design.

|  |  |
| ------------- | ------------- |
| Your Design  | <img width="941" alt="image" src="https://github.com/Eccb7/Portfolio2.0/assets/52934128/7ce2c1d8-31dd-46b8-9f71-71081ead414e">  |
| Figma Design  | ![image](https://github.com/Eccb7/Portfolio2.0/assets/52934128/dd9356c6-d57c-4451-b3c9-901ecef30062)  |

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![image](https://github.com/Eccb7/Portfolio2.0/assets/52934128/a88c27eb-e0b7-44b0-b261-884335d27ab1)  | ![image](https://github.com/Eccb7/Portfolio2.0/assets/52934128/e8b3bfb5-ec3a-413b-897b-99e83e397371)  |
| Content Cell  | Content Cell  |